qabelbox/src/test/java/de/qabel/qabelbox/chat/view/adapters/TextChatMessageViewHolderTest.kt

```diff
@@ -34,18 +37,21 @@ class TextChatMessageViewHolderTest {
         stub(contact.alias).toReturn("contact")
 
         val view = mock<View>()
+        stub(view.context).toReturn(RuntimeEnvironment.application)
```

Comment:
fyi.: afaik the `stub` method has been dropped. So this code will break as soon as you upgrade mockito above beta 2.0